tests/unit/test_old_sdk/test_clustering.py

```diff
@@ -51,6 +51,7 @@ def test_cluster_distribution(test_client: Client, clustered_dataset_id: List[Di
     assert True
 
 
+@pytest.mark.skip(reason="not rerouted lol")
```

Comment:
more professional reason pls xd

Reply:
THis wasn't re-routed in the gatway. Moving the skip in a future PR.